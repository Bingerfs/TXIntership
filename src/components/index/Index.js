import React, { Component } from 'react';

class Index extends Component{
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className='container' style={{height: '100vh'}}>
                <div className='row h-100'>
                    <div className='col-sm-12 my-auto d-flex justify-content-center'>
                        <ul>
                            <li>
                                <a href='/repositories'>Repositories</a>
                            </li>
                        </ul>
                    </div>
                </div>
            </div>
        );
    }
}

export default Index;