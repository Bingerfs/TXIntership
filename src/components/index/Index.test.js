import react from 'react';
import { shallow } from 'enzyme';
import Index from './Index.js';

describe('Index', () => {
    it("should render correctly", () => {
        const component = shallow(<Index />);
      });

    it("should render initial layout", () => {
        const component = shallow(<Index />);
        expect(component.getElements()).toMatchSnapshot();
    });
})