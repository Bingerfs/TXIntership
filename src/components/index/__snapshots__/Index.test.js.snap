// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render initial layout 1`] = `
Array [
  <div
    className="container"
    style={
      Object {
        "height": "100vh",
      }
    }
  >
    <div
      className="row h-100"
    >
      <div
        className="col-sm-12 my-auto d-flex justify-content-center"
      >
        <ul>
          <li>
            <a
              href="/repositories"
            >
              Repositories
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>,
]
`;
